modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/Helpers/JsonFromTokenUri.swift

```diff
@@ -23,6 +23,7 @@ final class JsonFromTokenUri {
     private let queue = DispatchQueue(label: "org.alphawallet.swift.jsonFromTokenUri")
```

Comment:
JsonFromTokenUri is thread safe, all calls perform on `queue`, so we just moved uri mapped to be called of that queue